R/expect_comparison_linter.R

```diff
@@ -6,6 +6,39 @@
 #'   also be used for such tests, but it is better to use the tailored function
 #'   instead.
 #'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "expect_true(x > y)",
+#'   linters = expect_comparison_linter()
+#' )
+#'
+#' lint(
+#'   text = "expect_true(x <= y)",
+#'   linters = expect_comparison_linter()
+#' )
+#'
+#' lint(
+#'   text = "expect_true(x == (y == 2))",
+#'   linters = expect_comparison_linter()
+#' )
+#'
+#' # okay
+#' lint(
+#'   text = "expect_gt(x, y)",
```

Comment:
to me a good "okay" example would be `expect_true(x < y | x > y^2)`

Reply:
I will add this as a separate "okay" example, but I will also retain the existing examples because I am trying to have a one-to-one mapping between code that produces lints and the same code modified not to produce a lint.

Reply:
👍 that was my intention, sorry not to be clearer